src/types.ts

```diff
@@ -635,6 +641,7 @@ export interface ManifestOptions {
    */
   scope_extensions: {
```

Comment:
should be (no jsdoc in the ScopeExtensionsType): 
```ts
export type ScopeExtensionsType = 'origin'
```

adding additional `@see` + default value in the entry:
```ts
/**
   * @see https://github.com/WICG/manifest-incubations/blob/gh-pages/scope_extensions-explainer.md
   * @see https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Manifest/Reference/scope_extensions
   */
  scope_extensions: {
    origin: string
    /**
      * @default 'origin'
      */
    type?: StringLiteralUnion<ScopeExtensionsType> | ScopeExtensionsType
  }[]